hw9/phone_book_json.py

```diff
@@ -0,0 +1,72 @@
+import json
+
+with open('json_pb.json', 'r+') as file_open:
```

Comment:
Add here a check if the file exists because otherwise your program fails if there is no file yet